site/content/faq/somethings-not-working-what-should-i-do.md

```diff
@@ -53,17 +53,17 @@ you.
 
 Be aware that lots of webapps do weird and wonderful things, often in bizarre ways. We can't second guess them, so when asking for help you'll need to give us sanitized examples to help us figure out what's going on.
 
-###  Ask on irc
+###  Ask on IRC
 
 Many of ZAP contributors hang out on the Libera Chat [#zaproxy](https://web.libera.chat/#zaproxy) channel which you can access via that link or your favourite irc client.
 
-###  Ask on twitter
+###  Ask on Twitter
 
 You can ask [@psiinon](https://twitter.com/psiinon) short questions directly
 on twitter, although whether he's online will be timezone and holiday
 dependant.
```

Comment:
Unrelated to title caps; this could be changed to "dependent".

Reply:
Sure